dialdbg/src/main.rs

```diff
@@ -197,10 +199,21 @@ async fn main() -> Result<()> {
             )?;
         log_config_setter = Some(log4rs::init_config(config)?);
 
-        dial_grpc(uri.as_str(), credential.as_str(), credential_type.as_str()).await;
+        let ch = dial_grpc(uri.as_str(), credential.as_str(), credential_type.as_str()).await;
         let grpc_res = parse::parse_grpc_logs(log_path.clone(), &mut out)?;
         write!(out, "{grpc_res}")?;
 
+        if let Some(ch) = ch {
+            if !args.nortt {
+                let average_rtt = rtt::measure_rtt(ch.clone(), 10).await?;
```

Comment:
(nit) I don't think this call to `clone()` is necessary?

Reply:
removed.